### PR TITLE
Add cron job to main

### PR DESCRIPTION
**Pull request by LaurieBennett**

### CODE REVIEW

**Comment by laurieboyes** [comment]
Ah easy as that eh? 😄 

You have a couple of conflicts though. Also am I right in thinking there were some changes you made in the keycloak UI that you needed to reflect in the terraform?

File: config/variables.tf [hunk 1]
```diff
@@ -26,4 +20,10 @@ variable "tf_url" {
   type        = string
   description = "The url of the keycloak to perform the terraform actions on"
   sensitive   = true
+}
+
+variable "cal_url" {
```
Comment (laurieboyes) [inline config/variables.tf]: Can’t spot where this is used, but if we still need it, please can we rename it for clarity?

```suggestion
variable "callisto_url" {
```

Comment (laurieboyes) [reply config/variables.tf]: Ah it was in the CSP right. Dang.

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -3,7 +3,9 @@ on:
   push:
     branches:
       - main
-      
+  schedule:
```
Comment (laurieboyes) [inline .github/workflows/main.yaml:7]: Something occurred to me recently - please could you add a bit to the readme that explains that we run this nightly and why?

Comment (LaurieBennett) [reply .github/workflows/main.yaml:7]: Done

File: config/variables.tf (hunk 1 above)
Comment (LaurieBennett) [reply config/variables.tf]: I've renamed the variable to something that's more immediately obvious

File: .github/workflows/main.yaml (hunk 2 above)
Comment (holly-blues) [reply .github/workflows/main.yaml:7]: Just second what LB mentioned as I'm not seeing a ReadMe

Comment (LaurieBennett) [reply .github/workflows/main.yaml:7]: Not sure why it wasn't showing up, I've also added the same schedule comment to validate.yaml

Comment (holly-blues) [reply .github/workflows/main.yaml:7]: Looks great!

**Pull request closed without merging**